src/lib/mdns/minimal/core/tests/TestQName.cpp

```diff
@@ -125,13 +125,61 @@ void ErrorTest(nlTestSuite * inSuite, void * inContext)
     }
 }
 
+void Comparison(nlTestSuite * inSuite, void * inContext)
+{
+    static const uint8_t kManyItems[] = "\04this\02is\01a\04test\00";
```

Comment:
You need a test for case-insensitive comparison.

Reply:
All my comparisons are cases sensitive. Are you saying the spec demands case insensitive? That is a bit painful if we need to fully support UTF-8. Hopefully we do not.